tests/test_time_series_graph.py

```diff
@@ -550,11 +550,23 @@ def test_from_adjacency_matrices(self):
         matrices[-1][0, 1] = 1
         # there should be 3+1 nodes
         variables = ['X1', 'X2', 'X3']
-        tsdag = TimeSeriesCausalGraph.from_adjacency_matrices(matrices, variables)
-        nodes = sorted(['X1', 'X2', 'X3', 'X1 lag(n=1)'])
-        self.assertEqual(sorted([n.identifier for n in tsdag.nodes]), nodes)
+        tsdag = TimeSeriesCausalGraph.from_adjacency_matrices(matrices, variables).get_minimal_graph()
+        nodes = sorted(['X2', 'X3', 'X1 lag(n=1)'])
+        self.assertEqual(set([n.identifier for n in tsdag.nodes]), set(nodes))
         self.assertEqual(len(tsdag.edges), 1)
 
+        full_insta = numpy.ones((3, 3))
+        numpy.fill_diagonal(full_insta, 0)
+        matrices = {0: full_insta, -1: numpy.zeros((3, 3))}
+        matrices[-1][0, 1] = 1
```

Comment:
What is this doing? Isn't it making one directed edge but we don't see that below. Worth checking it is there.